modules/ROOT/pages/configuration/connectors.adoc

```diff
@@ -1,34 +1,33 @@
-:description: How to configure connectors (Bolt, HTTP, and HTTPS) for Neo4j.
+:description: How to configure network connectors (Bolt, HTTP, and HTTPS) for Neo4j.
 [[connectors]]
-= Configure connectors
+= Configure network connectors
 
-Neo4j provides support for Bolt, HTTP, and HTTPS protocols using connectors.
-Connectors are configured in the _neo4j.conf_ file.
+Neo4j provides support for Bolt, HTTP, and HTTPS protocols using network connectors.
+Network connectors are configured in the _neo4j.conf_ file.
 
 [[connectors-available-connectors]]
-== Available connectors
+== Available network connectors
 
-The table below lists the available Neo4j connectors:
+The table below lists the available network connectors in Neo4j:
 
-.Neo4j connectors and port number
+.Neo4j network connectors and port number
 [options="header"]
 |===
-| Connector name  | Protocol | Default port number
+| Network connector name  | Protocol | Default port number
 | `server.bolt`   | Bolt     | `7687`
 | `server.http`   | HTTP     | `7474`
 | `server.https`  | HTTPS    | `7473`
 |===
 
-When configuring the HTTPS or Bolt connector, see also xref:security/ssl-framework.adoc[SSL framework] for details on how to work with SSL certificates.
+When configuring the HTTPS or Bolt connectors, see also xref:security/ssl-framework.adoc[SSL framework] for details on how to work with SSL certificates.
 
 
 [[connectors-configuration-options]]
 == Configuration options
 
-The connectors are configured by settings on the format `server.<connector-name>.<setting-suffix>>`.
-The available suffixes are described in the table below:
+The network connectors are configured by settings on the format `server.<network-connector-name>.<setting-suffix>>`.
```

Comment:
I think it should say: **in** the format

Reply:
yes, good spot